src/config/env.ts

```diff
@@ -64,7 +65,7 @@ export const SELFCARE_BASE_URL =
 
 export const STAGE = PAGOPA_ENV?.STAGE ?? 'DEV'
 
-export const SELFCARE_INTEROP_PROD_ID = `prod-interop${STAGE === 'UAT' ? '-coll' : ''}`
+export const SELFCARE_INTEROP_PROD_ID = getCurrentSelfCareProduct()
```

Comment:
I would remove it from the file at this point, and in case we need it I would directly call the function

Reply:
What do you think?